src/main/arm/parameters.json

```diff
@@ -38,12 +38,6 @@
         "appGatewaySSLCertPassword": {
             "value": "${appGatewaySSLCertPassword}"
         },
-        "keyVaultResourceGroup": {
-            "value": "${keyVaultResourceGroup}"
-        },
-        "keyVaultName": {
-            "value": "${keyVaultName}"
-        },
         "keyVaultSSLCertDataSecretName": {
             "value": "${keyVaultSSLCertDataSecretName}"
         },
```

Comment:
Is the secret name kept for the auto-generated self-signed certificate?

Reply:
Good catch, fixed in commit [1813a8d](https://github.com/WASdev/azure.liberty.aks/pull/124/commits/1813a8d41718fa41a0b85070ec140e3526e416e6), which removes both unused parameters `keyVaultSSLCertDataSecretName` and `keyVaultSSLCertPasswordSecretName`.